incubator/starter/README.md

```diff
@@ -1,8 +1,50 @@
 # Starter Stack
 
-A minimal, runnable starter stack that can be used as a base for developing new stacks. This stack provides a sample application template consisting of single line bash scripts, along with all the settings in Dockerfiles and manifests to allow you to use the Appsody CLI to test out init/run/debug/test/build/deploy ahead of starting to make changes to a copy of this stack for yourself.
+A minimal, runnable starter stack that can be used as a base for developing new stacks. This stack provides a sample application template consisting of single line bash scripts, along with all the settings in Dockerfiles and manifests to allow you to use the Appsody CLI to test out package/init/run/debug/test/build/deploy ahead of starting to make changes to a copy of this stack for yourself.
 
-For any new stack, this Readme should explain in detail about this stack and any templates included, e.g.:
+To make a copy of this stack as a basis for your new stack, enter, for example:
 
-- How can a user start using this stack for application development
+```bash
+$ cd ~
+$ appsody stack create mystack --copy incubator/starter
+$ cd mystack
+$ ls - al
+total 16
+drwxr-xr-x  6 henrynash  staff  192 21 Oct 00:14 .
+drwxr-xr-x  3 henrynash  staff   96 21 Oct 00:14 ..
+-rw-r--r--  1 henrynash  staff  621 21 Oct 00:14 README.md
+drwxr-xr-x  7 henrynash  staff  224 21 Oct 00:14 image
+-rw-r--r--  1 henrynash  staff  297 21 Oct 00:14 stack.yaml
+drwxr-xr-x  3 henrynash  staff   96 21 Oct 00:14 templates
+```
+
+The initial version of your new stack is now ready to be packaged (using `appsody stack package`) and will then be available to the Appsody CLI. You can check that this is working, ahead of starting to modify the stack structure to match the requirements of your new stack:
+
+```bash
+$ appsody stack package
+$ appsody list dev-local
+REPO             ID       VERSION         TEMPLATES       DESCRIPTION
+dev-local        mystack  0.1.0           *simple         sample stack to help...
+```
+
+Acting as an application developer, you can now use the CLI to initialize a project using this stack:
+
```

Comment:
We could mention setting `APPSODY_PULL_POLICY=IFNOTPRESENT` here, as we want to use the locally cached image

Reply:
Yes, I do that in the main tutorial, but might be good to add here as well. I will push a new version of the PR that has this.